### PR TITLE
Allow for skipping SSL certificate validation

### PR DESCRIPTION
**Pull request by bboozzoo**
It might be useful to skip SSL certificate validation when troubleshooting integration issues.

@kacf @pasinskim

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -132,6 +133,7 @@ func argsParse(args []string) (runOptionsType, error) {
 			*certKey,
 			*serverCert,
 			false,
+			*skipVerify,
```
Comment (kacf) [inline main.go]: I suggest using labels here, there are quite a few entries, including two bools right next to each other. If we somehow mess up the order later on, the compiler won't warn you as is.

File: config.go [hunk 2]
```diff
@@ -79,6 +80,7 @@ func (c menderConfig) GetHttpConfig() httpsClientConfig {
 		c.HttpsClient.Key,
 		c.ServerCertificate,
 		c.ClientProtocol == "https",
+		c.HttpsClient.SkipVerify,
```
Comment (kacf) [inline config.go]: I suggest labels here as well.

**Comment by kacf** [comment]
Otherwise +1 from me!

File: main.go (hunk 1 above)
Comment (bboozzoo) [inline main.go]: Makes sense, will do that.

Comment (bboozzoo) [inline main.go]: fixed

File: config.go (hunk 2 above)
Comment (bboozzoo) [inline config.go]: fixed

**Comment by kacf** [comment]
trigger build

**Pull request merged**